src/test/objectstore/store_test.cc

```diff
@@ -5792,8 +5792,12 @@ void doMany4KWritesTest(boost::scoped_ptr<ObjectStore>& store,
     test_obj.write();
   }
   test_obj.wait_for_done();
-  if (res_stat) {
-    test_obj.statfs(*res_stat);
+  test_obj.statfs(res_stat);
+  if (!(res_stat.stored <= max_object_size) ||
+      !(res_stat.allocated <= max_object_size)) {
```

Comment:
shall we put
```c++
if (res_stat.stored > max_object_size ||
    res_stat.allocated != max_object_size) {
```
instead?

Reply:
IMO - No - as the original condition (_EQ) in asserts isn't that correct - potentially we can have less allocations than the max obj size - if we do just a few writes or random generation was not good enough. This have small probability at the moment but we can probably start observing some errors if parameter changes..